test/valid_field_test.exs

```diff
@@ -24,7 +24,7 @@ defmodule ValidFieldTest do
       |> ValidField.assert_invalid_field(:title, ["", nil, "Something else"])
     end
 
-    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<2016-08-23>", fn ->
+    assert_raise ValidField.ValidationError, "Expected the following values to be invalid for \"date_of_birth\": #Ecto.Date<#{get_current_date}>", fn ->
```

Comment:
if `get_current_date/0` is not being used elsewhere you should just interpolate on `Ecto.Date.utc()`